pkg/config/config_template.yaml

```diff
@@ -1265,6 +1265,10 @@ api_key:
   ##        Enables caching obfuscated statements. Currently supported for SQL and MongoDB queries.
   ##        Enabled by default.
   #         enabled: true
+  ##        @param DD_APM_OBFUSCATION_CACHE_MAX_SIZE - integer - optional - default: 5000000
+  ##        The maximum size of the cache in bytes. The maximum allowed resource length is 5000.
```

Comment:
I'm not sure what the "resource length" is, so you might want to explain it

Reply:
This doc explains it!

https://github.com/DataDog/documentation/blob/8ba05a9cebbe6f61da29fe80a9456df30b528822/content/en/tracing/guide/send_traces_to_agent_by_api.md?plain=1#L57